tests/ui/async-await/issues/issue-78938-async-block.stderr

```diff
@@ -1,12 +1,10 @@
 error[E0373]: async block may outlive the current function, but it borrows `room_ref`, which is owned by the current function
   --> $DIR/issue-78938-async-block.rs:8:33
    |
-LL |       let gameloop_handle = spawn(async {
-   |  _________________________________^
-LL | |         game_loop(Arc::clone(&room_ref))
-   | |                               -------- `room_ref` is borrowed here
-LL | |     });
-   | |_____^ may outlive borrowed value `room_ref`
```

Comment:
Here's the only place that the diagnostic arguably got worse. I kinda don't think it matters, though, because the primary message still says:

> async block may outlive the current function, but it borrows `room_ref`, which is owned by the current function

So it's still clear that the problem is with an async block.

If you care enough about this for it to get fixed, then I suggest we change the label to state *what* may outlive the borrowed value, i.e.:

```
LL |     let gameloop_handle = spawn(async {
   |                                 ^^^^^ async block may outlive borrowed value `room_ref`
```